include/NovelRT/NovelRunner.h

```diff
@@ -41,18 +47,24 @@ namespace NovelRT {
      * Starts the visual novel.
```

Comment:
lol I've just seen this - could you update this to specify that it launches NovelRT/the NovelRT game loop/the game instead? Much love :D

Reply:
Sure 👌👌